apps/jobs/urls.py

```diff
@@ -3,9 +3,9 @@
 from . import views
 
 urlpatterns = [
-    url(r'challenge/(?P<challenge_id>[0-9]+)/'
```

Comment:
I am really not able to infer any change in `jobs` app. The only valid thing that I could infer was adding support for `PUT` method, but there is no action for the same method. 

If that is the case(PUT method), please only add that change in this PR and the cleaning and updating part can go in other PR?


Also are we not following of creating different PR for frontend and backend changes?